dpanonymize/__init__.py

```diff
@@ -17,17 +18,30 @@
     'mri': MRI,
     'interviews': VIDEO,
     'audio': AUDIO,
-    'video': VIDEO
+    'video': VIDEO,
+    'eeg': EEG
 }
 
 
 class FileInPhoenixBIDS(object):
-    '''PHOENIX file class used to grab file information'''
-    def __init__(self, file_path):
+    '''PHOENIX file class used to grab file information
+
+    Key Arguments:
+        - file_path: file path under BIDS phoenix structure, str
+          Example file_path
```

Comment:
I feel like this slide should be attached to every sentence related to BIDS:

https://github.com/AMP-SCZ/lochness/pull/40#discussion_r671856493

Reply:
Yeap you've pointed out that we should do this. I'll add these doc strings to places when I work on another branch of cleaning things up, after we have the fully working pipeline. Thanks.